src/main/java/seedu/address/model/task/Task.java

```diff
@@ -0,0 +1,133 @@
+package seedu.address.model.task;
+
+import static java.util.Objects.requireNonNull;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import seedu.address.model.employee.Employee;
+
+/**
+ * Represents a task which belongs to a certain project.
+ */
+public class Task {
+
+    public static final String MESSAGE_CONSTRAINTS = "Tasks must be bound to a project first"
+        + " and later assigned to an Employee";
+
+    /**
+     * The first character must not be a whitespace,
+     * otherwise " " (a blank string) becomes a valid input.
+     */
+    public static final String VALIDATION_REGEX = "[^\\s].*";
+    private final String name;
+    private boolean isDone;
+    private Employee employee;
+    private final LocalDateTime deadline;
```

Comment:
What is the design choice here to not create classes for each attribute in `Task` model? (Since `Employee` and `Project` models create classes for each attribute)